app/views/services/_connection_menu.html.erb

```diff
@@ -68,7 +80,7 @@
           <li data-page-type="singlequestion"
               data-component-type="textarea"><a href="#add-page">Textarea</a></li>
           <li data-page-type="singlequestion"
-              data-component-type="email"><a href="#add-page">Email address</a></li>
+              data-component-type="mail"><a href="#add-page">Email address</a></li>
```

Comment:
in terms of what the backend knows this component as it is `email`. does the frontend use this differently?

Reply:
No, this looks like I've mistakenly managed to remove the e 🤦‍♂️